connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerSourceTaskTest.java

```diff
@@ -68,7 +65,6 @@
 
 @RunWith(PowerMockRunner.class)
 public class WorkerSourceTaskTest extends ThreadedTest {
```

Comment:
Actually, there are a few other tests that use this too. I'll file a JIRA to clean it up -- it's not reliable enough anyway (and maybe some of the tests can just drop the use of the thread and mock the necessary bits instead if that doesn't make things even messier).